src/intern.rs

```diff
@@ -104,13 +103,13 @@ pub struct Interner<T: Hash + Eq> {
 }
 
 impl<T: Hash + Eq> Interner<T> {
-    /// Create an Interner with an intial capacity calculated by calling
+    /// Create an Interner with an initial capacity calculated by calling.
```

Comment:
not sure that there should be a period here

Reply:
Right, it's not actually complete. Please feel free to finish it ,maybe then I can merge as well. Pushing into this branch should suffice.

Reply:
@loftyinclination Could you make the changes? It really needs another commit that isn't mine so I can merge. Thank you.

Reply:
feel free to push yourself @Byron  and I can merge

Reply:
sorry for the delay, just pushed